src/rockstor/fs/tests/test_btrfs.py

```diff
@@ -10,10 +10,12 @@
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 General Public License for more details.
 You should have received a copy of the GNU General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
+along with this program. If not, see <https://www.gnu.org/licenses/>.
 """
 import json
 import unittest
+from unittest import mock
+from unittest.mock import patch
```

Comment:
@phillxnet ,
I'll show my lack of understanding here but: is this import necessary if we do use `mock.patch()` and `mock.patch.stopall()` for instance?

Reply:
I had thought the same actually but it seems PyCharm was a little confused. Until I did this explicitly there were a number of flags. I'm keen to merge this so that I can approach the linked issue. I'll retrofit if all looks OK with the other 30 or so related changes.